src/pudl/analysis/plant_parts_eia.py

```diff
@@ -309,6 +309,18 @@
     "operational_status": ["existing", "proposed", "retired"],
 }
 
+MAX_MIN_ATTRIBUTES_DICT = {
```

Comment:
I reworked this so it's kind of a dictionary of parameters for each added attribute. Idk if this is actually cleaner.